bin/common/start_docker_service.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+sed -i 's/DOCKER_OPTS="\(.*\)"/DOCKER_OPTS="\1 --storage-driver=overlay2"/' /etc/sysconfig/docker
```

Comment:
Right, but there's no `--host $IP` here at least.  I guess you inherited bad code from me :)